tests/system/datastore/test_google_cloud_storage.py

```diff
@@ -66,7 +66,7 @@ def setup_class(cls):
             token = credentials
         except json.JSONDecodeError:
             token = cls.credentials_path
-        cls._gcs_fs = fsspec.filesystem("gcs", token=token)
+        cls._gcs_fs = fsspec.filesystem("gcs", token=token, use_listings_cache=False)
```

Comment:
Need to add a comment explaining why this is there, including the version this became necessary.